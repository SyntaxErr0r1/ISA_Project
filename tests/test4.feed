http://localhost:42103/www.theregister.com.xml
http://localhost:42103/what-if.xkcd.com.xml
http://localhost:42103/en.wikipedia.org.xml