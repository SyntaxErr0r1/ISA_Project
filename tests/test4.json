{
    "args": "-f tests/test4.feed",
    "feedFile": [
        "www.theregister.com.xml",
        "what-if.xkcd.com.xml",
        "en.wikipedia.org.xml"
    ],
    "returnCode": 0
}