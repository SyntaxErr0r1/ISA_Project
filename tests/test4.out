*** The Register ***
AMD grows Epyc datacenter share, loses to Intel generally

Instagram star gets 11 years for $300m email scam plot

TSMC: You know what would be fab? Some local neon supplies

AMD's 96-core Epyc CPUs leapfrog Intel to put DDR5, PCIe 5.0 in the datacenter

No formal certifications? CUE the Ubuntu skills testing scheme

Tesla rival Rivian posts losses of $1.7b, with worse to come

Husband and wife nuclear warship 'spy' team get 20 years each

Twitter Chief Information Security Officer flies the coop

SAP's vision underplays complexity of S/4HANA cloud migration, says Gartner

Apple and Amazon conspired to raise iPhone and iPad prices, claims class action lawsuit

EU set to sign internet satellite deal, as UK frees up spectrum

The Osprey has landed: IBM's 433-qubit quantum processor

NTT claims it can stop the noise leaking from annoying people's headphones

OpenPrinting keeps old printers working – even on Windows

CIOs in Europe warned: Be wary of tech price inflation

Microsoft's grand unified theory of .NET advances a little

Look! Up in the sky! Proof of concept for satellites beaming energy to Earth!

I'm happy paying Twitter eight bucks a month because price isn't the same as value

Cygnus cargo ship makes it to ISS with blanketed solar panel

Windows breaks under upgraded IceXLoader malware

Musk sows more Twitter chaos, now with Official policy snafu

Republican senators tell FTC to back off data security, surveillance rules

Theranos founder Elizabeth Holmes's arguments for new trial deemed spurious – just like her tech

Wells Fargo, Zelle slammed by Liz Warren over rampant online banking fraud

Mythic bet big on analog AI but has run out of cash

Optimizer Rescale recommends Rescale's optimization recommender

GlobalFoundries expects semi market dip – but only in first half of 2023

Feel Luckey, punk? Oculus designer builds VR murder headset

TSMC reportedly looks to raise a second Arizona chip fab

Musk sells $3.95 billion in Tesla shares, paid eleven times more for Twitter

Salesforce trims workforce as growth slows post-lockdowns

Intel takes on AMD and Nvidia with mad 'Max' chips for HPC

HPE goes Cray for Intel's Sapphire Rapids Xeons in latest supers

Unlucky for some: Meta chops 13% of global workforce

China's first domestic single-aisle jet, the C919, scores 300 orders

Strong support for Snap and Ubuntu Core as Canonical meet IRL

Ampere says no changes to its Arm licensing as it readies new chips

Microsoft's $69B deal to buy Activision Blizzard under investigation by EU regulators

This ancient quasar may be the remains of the first-gen star that started us all

Boffins find COVID changed the way sysadmins work – probably for the worse

Euro clouds lodge another complaint against Microsoft over anti-competitive licenses

LG debuts thin malleable screens made from contact lens material

IBM so confident in supply chain, it's frozen storage prices and has freebies for late shipments

Zoom adds email and calendar to its apps, to relieve the crushing burden of ALT-TAB

VMware warns of three critical holes in remote-control tool

Microsoft squashes six security bugs already exploited in the wild

With potential hurricane approaching, NASA leaves mega-rocket on launch pad

Marvell's latest switch may not be fast but the timing is great

Intel's top-spec Raptor Canyon NUC can double as a 700+W space heater

Swiss Re wants government bail out as cybercrime insurance costs spike

*** what if? ***
Transatlantic Car Rental

Hailstones

Hot Banana

Earth-Moon Fire Pole

Electrofishing for Whales

*** Wikipedia  - Recent changes [en] ***
Battery Ventures

User:Jonesey95/sandbox

Eastern Mountain Sports

